src/cli/cli_commissioner.cpp

```diff
@@ -40,6 +40,24 @@
 namespace ot {
 namespace Cli {
 
+/**
```

Comment:
We probably need to note that these commands are only available when OPENTHREAD_CONFIG_COMMISSIONER_ENABLE and OPENTHREAD_FTD are set (see line 38)

Reply:
@Vyrastas - thx, I will add this to the commissioner start command